tests/test_filters.py

```
@@ -0,0 +1,234 @@
+        assert should is False
+        assert decision.reason == "ignore_pattern_match"
+
+    def test_strict_mode_rejects_noise(self, monkeypatch):
+        monkeypatch.setenv("MNEMOSYNE_WRITE_CLASSIFIER", "strict")
+        should, decision = should_remember("$ pip install foo\nCollecting foo")
+        assert should is False
+        assert decision.action == "reject"
+
+    def test_strict_mode_allows_valuable(self, monkeypatch):
+        monkeypatch.setenv("MNEMOSYNE_WRITE_CLASSIFIER", "strict")
+        should, decision = should_remember("User prefers pytest for testing.")
```

Ruff (0.15.20)

[warning] Unpacked variable `decision` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)